cmd/admin/pause.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/letsencrypt/boulder/identifier"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+// subcommandPauseBatch encapsulates the "admin pause-batch" commands.
+type subcommandPauseBatch struct {
+	file string
+}
+
+var _ subcommand = (*subcommandPauseBatch)(nil)
+
+func (p *subcommandPauseBatch) Desc() string {
+	return "Batch pause a CSV containing (account, identifier type, list of identifier strings)"
+}
+
+func (p *subcommandPauseBatch) Flags(flag *flag.FlagSet) {
+	flag.StringVar(&p.file, "file", "", "Path to CSV file containing (account, identifier type, list of identifier strings)")
+}
+
+func (p *subcommandPauseBatch) Run(ctx context.Context, a *admin) error {
+	if p.file == "" {
+		return errors.New("the -file flag is required")
+	}
+
+	identifiers, err := a.readPausedAccountFile(p.file)
+	if err != nil {
+		return err
+	}
+
+	err = a.pauseIdentifiers(identifiers)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// subcommandUnpauseBatch encapsulates the "admin unpause-batch" commands.
+type subcommandUnpauseBatch struct {
+	file string
+}
+
+var _ subcommand = (*subcommandUnpauseBatch)(nil)
+
+func (u *subcommandUnpauseBatch) Desc() string {
+	return "Batch unpause a CSV containing (account, identifier type, list of identifier strings)"
```

Comment:
Why does this input CSV have columns for the identifier type and value, when we only unpause entire accounts? I think this input file should just be a list of regIDs.

Reply:
I can clean this up. For this first pass I just wanted to make the minimum functionally viable product.